src/containers/Profile.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <div class="profileContainer">
+    <div class="profileContainer-header">
+      <div class="profileContainer-title">
+        <span> Edit Profile</span>
+      </div>
+      <div class="profileContainer-subtitle">
+        <span> get your information up-to-date! </span>
+      </div>
+    </div>
+    <div class="profileContainer-editPage">
+      <onboarding-basic
+        :userName="name"
+        :onboardingData="onboarding"
+        :isEditingProfile="true"
+        @updateBasic="updateBasic"
+      />
+    </div>
+    <div>
+      <button
+        class="profileContainer-button"
+        @click="submitOnboarding"
+        data-cyId="onboarding-finishButton"
+      >
+        Finish
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { PropType, defineComponent } from 'vue';
+import OnboardingBasic from '@/components/Modals/Onboarding/OnboardingBasic.vue';
+import { setAppOnboardingData } from '@/global-firestore-data';
+import { getMajorFullName, getMinorFullName, getGradFullName } from '@/utilities';
+import timeline1Text from '@/assets/images/timeline1text.svg';
+import timeline2Text from '@/assets/images/timeline2text.svg';
+import timeline3Text from '@/assets/images/timeline3text.svg';
+
+const timelineTexts = [timeline1Text, timeline2Text, timeline3Text];
+
+const placeholderText = 'Select one';
+const FINAL_PAGE = 3;
+
+export default defineComponent({
+  components: { OnboardingBasic },
+  props: {
+    userName: { type: Object as PropType<FirestoreUserName>, required: true },
+    onboardingData: {
+      type: Object as PropType<AppOnboardingData>,
+      required: true,
+    },
+  },
+  emits: ['onboard', 'cancelOnboarding'],
+  data() {
+    return {
+      currentPage: 1,
+      name: { ...this.userName },
+      onboarding: { ...this.onboardingData },
+    };
+  },
+  computed: {
+    // Display error if a required field is empty
+    isError(): boolean {
+      return (
+        this.name.firstName === '' ||
+        this.name.lastName === '' ||
+        this.onboarding.gradYear === '' ||
+        !this.onboarding.gradSem ||
+        this.onboarding.entranceYear === '' ||
+        !this.onboarding.entranceSem ||
+        (this.onboarding.college === '' && this.onboarding.grad === '')
+      );
+    },
+    timelineTextImage(): string {
```

Comment:
I don't think this computed method is being called anywhere in this new container. Could you remove it along with the imported timeline svgs?

Reply:
This comment applies to a lot of things in this file - such as the `FINAL_PAGE` constant, the emits, etc. Could you clean it up a little? You can always add back stuff as needed.